app/ui/components/categoryHomePage.tsx

```diff
@@ -0,0 +1,156 @@
+        <GridCategory
+          gridBackground="#FFE8E8"
+          gridHeight="350px"
+          gridWidth="340px"
+          gridRadius="20px"
+          gridBorder="white 9px solid"
+          gridBoxShadow="lg"
+          imageAlt="image zeby"
+          imageSrc="Gridlistimage.png"
+          boxTextAlign="center"
+          boxBackgroud="white"
+          boxRadiusTop="25px"
+          fontWeight="400"
+          fontSize="2xl"
+          fontFamilly="Inter"
+          lineHeight="120%"
+          letterSpacing="-1px"
+          textValue="Scrum Master"
+          buttonTextColor="white"
+          buttonBackground="#2945FF"
+          buttonSize="lg"
+          buttonRadius="100px"
+          buttonContent="Découvrir"
+        />
+        <GridCategory
+          gridBackground="#FFE8E8"
+          gridHeight="350px"
+          gridWidth="340px"
+          gridRadius="20px"
+          gridBorder="white 9px solid"
+          gridBoxShadow="lg"
+          imageAlt="image zeby"
+          imageSrc="Gridlistimage.png"
+          boxTextAlign="center"
+          boxBackgroud="white"
+          boxRadiusTop="25px"
+          fontWeight="400"
+          fontSize="2xl"
+          fontFamilly="Inter"
+          lineHeight="120%"
+          letterSpacing="-1px"
+          textValue="Scrum Master"
+          buttonTextColor="white"
+          buttonBackground="#2945FF"
+          buttonSize="lg"
+          buttonRadius="100px"
+          buttonContent="Découvrir"
+        />
+        <GridCategory
+          gridBackground="#FFE8E8"
+          gridHeight="350px"
+          gridWidth="340px"
+          gridRadius="20px"
+          gridBorder="white 9px solid"
+          gridBoxShadow="lg"
+          imageAlt="image zeby"
+          imageSrc="Gridlistimage.png"
+          boxTextAlign="center"
+          boxBackgroud="white"
+          boxRadiusTop="25px"
+          fontWeight="400"
+          fontSize="2xl"
+          fontFamilly="Inter"
+          lineHeight="120%"
+          letterSpacing="-1px"
+          textValue="Scrum Master"
+          buttonTextColor="white"
+          buttonBackground="#2945FF"
+          buttonSize="lg"
+          buttonRadius="100px"
+          buttonContent="Découvrir"
+        />
+        <GridCategory
+          gridBackground="#FFE8E8"
+          gridHeight="350px"
+          gridWidth="340px"
+          gridRadius="20px"
+          gridBorder="white 9px solid"
+          gridBoxShadow="lg"
+          imageAlt="image zeby"
+          imageSrc="Gridlistimage.png"
+          boxTextAlign="center"
+          boxBackgroud="white"
+          boxRadiusTop="25px"
+          fontWeight="400"
+          fontSize="2xl"
+          fontFamilly="Inter"
+          lineHeight="120%"
+          letterSpacing="-1px"
+          textValue="Scrum Master"
+          buttonTextColor="white"
+          buttonBackground="#2945FF"
+          buttonSize="lg"
+          buttonRadius="100px"
+          buttonContent="Découvrir"
+        />
```

Comment:
All the iteration of this should be an Import of a component since its duplicata (this is a good practicesince you'll define the common style element of all the cards inside this section)